README.md

```diff
@@ -21,7 +21,7 @@ A library to describe **Typed Condensed Oriented Directed Acyclic Planar Multigr
 | **Condensed Graph** | A graph where every pair of vertices are connected by either zero or one edge |
 | **Multigraphs** | A graph without loops |
 | **Oriented Graph** | A graph without any symmetric pair of directed edges |
-| **Typed Graph** | Not a term from Graph Theory but from Computer Science. A typed graph is a graph where each edges and nodes have a given data type. |
+| **Typed Graph** | Not a term from Graph Theory but from Computer Science. A typed graph is a graph where each one of the edges and the nodes has a given data type. |
```

Comment:
why only `one of the edges` ?

Reply:
it's more "each one of the edges", because "each edges and nodes" doesn't make much sense

Reply:
I like the @fakirAyoub clarification better.